x-pack/plugins/enterprise_search/server/utils/fetch_enterprise_search.test.ts

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import '../__mocks__/http_agent.mock';
+
+jest.mock('node-fetch');
+import fetch from 'node-fetch';
+
+import { loggingSystemMock } from '@kbn/core-logging-server-mocks';
+import { KibanaRequest } from '@kbn/core/server';
+
+import { ConfigType } from '..';
+
+import { fetchEnterpriseSearch, isResponseError } from './fetch_enterprise_search';
+
+describe('fetchEnterpriseSearch', () => {
```

Comment:
I stopped myself from writing more tests here since I think we'll end up removing this when migrate engines over to the ES module